react/features/prejoin/functions.js

```diff
@@ -25,6 +25,16 @@ export function isDeviceStatusVisible(state: Object): boolean {
     && !state['features/base/config'].startSilent;
 }
 
+/**
+ * Selector for determining if the display name is mandatory.
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {boolean}
+ */
+export function isDisplayNameRequired(state: Object): boolean {
+    return state['features/prejoin'].isDisplayNameRequired;
```

Comment:
Also check for this config option: https://github.com/jitsi/jitsi-meet/blob/ea2ea89ef76e36505f2f1114f12c1917a91f6803/config.js#L267